### DISCUSSION TITLE
mxplayer codec vs namida

### DISCUSSION THREAD

**Discussion by naderalipour** [namidaco/namida#341] [General]
mxplayer codec vs namida.half files are missing.
![Screenshot_2024-09-05-10-04-24-867_com mxtech videoplayer pro](https://github.com/user-attachments/assets/e99bf347-2293-4b6d-ab38-80e63349907f)
![Screenshot_2024-09-05-10-02-27-790_com msob7y namida](https://github.com/user-attachments/assets/612fe132-5b00-4fd7-9a6e-1371265655d3)
![Screenshot_2024-09-05-10-02-19-146_com mxtech videoplayer pro](https://github.com/user-attachments/assets/e8713fea-a7b0-4a55-8f71-7ef188992271)
![Screenshot_2024-09-05-10-07-26-017_com msob7y namida](https://github.com/user-attachments/assets/ee532230-7988-4fec-972f-53cc08359f02)

**Comment by VerySweetBread** [comment]
It doesn't show your file system, it has indexer and show only tracked files/folders

**Comment by MSOB7YY** [comment]
1. codec has nothing to do with this
2. mxplayer shows ur video only
3. you are comparing namida showing audio files to mxplayer showing video files
4. @VerySweetBread mxplayer doesnt have filesystem too, they rely on indexer but difference is that it allows choosing root folder + automatically refresh
5. since namida [beta v4.0.0](github.com/namidaco/namida-snapshots/releases), videos can be indexed too, they will have a separate folder view
6. extra: u can disable folders hierarchy in extra settings to see all folders at once

**Reply by naderalipour** [reply]
It cannot finish searching on beta.still 99%.i deleted this massive junkfiles folder.tnx to folder scan view.folders at onece (hierarchy in extra)doesn't work also.

**Reply by naderalipour** [reply]
File Folder size.pls.

**Reply by naderalipour** [reply]
Folder mode has not sorting option.

**Reply by MSOB7YY** [reply]
sorry i can't understand anything, please elaborate more on whats the issue encountering you
